__tests__/app.test.js

```diff
@@ -0,0 +1,24 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data");
+
+afterAll(() => db.end());
+beforeEach(() => seed(testData));
+
+describe("GET api/categories", () => {
+  test("200: should succesfully get categories", () => {
+    return request(app)
+      .get("/api/categories")
+      .expect(200)
+      .then(({ body: categories }) => {
```

Comment:
Nice assertion below, the only hole in the testing is that if your endpoint gave back a empty arr for whatever reason it would also pass the test even thought it's wrong. 
We always want to supplement our assertion of the shape of the array of objs (below) with a length check to make sure it is an array of the right num of items

Reply:
Ahhhh okay, thank you. Do i merge after i complete my test or do i send in another nchelp and then awaiting response?